### PR TITLE
Exercise/raindrops

### PR DESCRIPTION
**Pull request by GroophyLifefor**
New exercise implementation.

https://github.com/exercism/problem-specifications/blob/main/exercises/raindrops/instructions.md

### CODE REVIEW

**Comment by GroophyLifefor** [comment]
I saw right now, I have to add newline at end each new file.

File: exercises/practice/raindrops/raindrops.bat
Comment (ErikSchierboom) [inline exercises/practice/raindrops/raindrops.bat:1]: To be consistent, this file should probably be called `Raindrops.bat`

File: exercises/practice/raindrops/raindropsTest.bat
Comment (ErikSchierboom) [inline exercises/practice/raindrops/raindropsTest.bat:1]: To be consistent, this file should probably be called `RaindropsTest.bat`

**Review by ErikSchierboom — APPROVED** [review]

**Comment by GroophyLifefor** [comment]
@ErikSchierboom Hi Erik, thanks for your review. Can you check is there any other problem?

**Comment by ErikSchierboom** [comment]
@GroophyLifefor I don't see the names of the file being changed. You might need to do `git mv <old_filename> <new_filename>` for the names to actually change.

**Comment by GroophyLifefor** [comment]
@ErikSchierboom I do in this comment right now https://github.com/exercism/batch/commit/c51e281f45a0ccfb22176f3b7730aea1f8a13039 but doesn't affected to this PR, I'm not sure why.

**Comment by ErikSchierboom** [comment]
I can look into it next week. I'll approve

**Comment by ErikSchierboom** [comment]
Could you look at failing ci?

**Comment by GroophyLifefor** [comment]
@ErikSchierboom CI/CD fixed.

**Comment by ErikSchierboom** [comment]
Feel free to merge!

**Pull request merged**